docs/source/user_guide/supported_features.md

```diff
@@ -0,0 +1,20 @@
+# Supported Features
+
+| Feature                       | V0 | V1 |
+|-------------------------------|----|----|
+| Quantization                  |    |✅︎  |
+| LoRA                          |⚠️  |    |
+| Tensor Parallel               |✅︎  |✅︎  |
+| Prompt Adapter                |✅︎  |✅︎  |
+| Guided Decoding               |✅︎  |✅︎  |
+| Speculative Decoding          |    |✅︎  |
```

Comment:
I'm at least 50% certain that this would have to be implemented in the model runner, and we have not hooked it up. We always advance one token at a time, right?